array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,8 @@
+/* exported dropRight */
+function dropRight(array, count) {
+  var newArray = [];
+  for (var i = 0; i < array.length - count && array.length > 0; i++) {
```

Comment:
Is the second condition in this really necessary? If the length is `0`, then the `0` value of `i` would be greater than the negative number, so the loop wouldn't start anyway. Try running this without that second condition, and see if it works.